index.html

```diff
@@ -22,13 +22,14 @@
       <div class="row">
         <div class="column-full">
           <h1 class="journal-header">Code Journal</h1>
-          <a href="#">
+          <a class="entries-link" href="#">
             <h1 class="entries">Entries</h1>
```

Comment:
You can only have 1 `h1` element on a page, or else it gets flagged as an accessibility problem. I would just put the text directly inside the `a` tag, or use a different element like `span` inside the `a`